com.biglybt.BiglyBT.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: com.biglybt.BiglyBT
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: biglybt-runner
+separate-locales: false
+finish-args:
+  - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --persist=.biglybt
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: biglybt
+    buildsystem: simple
+    build-commands:
+      - install -dm755 /app/bin
+      - install -dm755 /app/biglybt
+      - mv * /app/biglybt/
+      - chmod -R a-s,go+rX,go-w /app/biglybt
+      - install -Dm644 /app/biglybt/biglybt_512.png /app/share/icons/hicolor/512x512/apps/com.biglybt.BiglyBT.png
+      - install -Dm644 /app/biglybt/biglybt.png /app/share/icons/hicolor/256x256/apps/com.biglybt.BiglyBT.png
+      - install -Dm644 /app/biglybt/biglybt_128.png /app/share/icons/hicolor/128x128/apps/com.biglybt.BiglyBT.png
+      - install -Dm644 /app/biglybt/biglybt_64.png /app/share/icons/hicolor/64x64/apps/com.biglybt.BiglyBT.png
+      - install -Dm644 /app/biglybt/biglybt_32.png /app/share/icons/hicolor/32x32/apps/com.biglybt.BiglyBT.png
+      - install -Dm644 /app/biglybt/biglybt_16.png /app/share/icons/hicolor/16x16/apps/com.biglybt.BiglyBT.png
+      - install -Dm644 /app/biglybt/com.biglybt.BiglyBT.metainfo.xml /app/share/metainfo/com.biglybt.BiglyBT.metainfo.xml
+      - install -Dm644 /app/biglybt/com.biglybt.BiglyBT.desktop /app/share/applications/com.biglybt.BiglyBT.desktop
+      - install -m755 /app/biglybt/biglybt-runner /app/bin/biglybt-runner
```

Comment:
this would be better if you used `dest` for the main archive (and changed the `mv`) or if you didn't `mv *`

Reply:
used dest